client/state/admin-menu/fallback-data.json

```diff
@@ -0,0 +1,479 @@
+[
+    {
+        "icon": "dashicons-feedback",
+        "slug": "feedback",
+        "title": "Feedback",
+        "type": "menu-item",
+        "url": "https://wordpress.com/wp-admin/?page=feedback",
```

Comment:
URLs for the fallback should probably default to Calypso

Reply:
Especially this one doesn't have a Calypso equivalent ;-). I am more in favor of replicating the `wp-admin` URLs status https://docs.google.com/spreadsheets/d/1i640dQt4vs2XYgqRA4YEH69ssR9UNcFJx49HUss0-q0/edit#gid=0 . That way, they will actually miss only the custom plugins registered menus? Should we have a different fallback for jetpack only connected sites as they will have a smaller set of menus eg import / export are redirecting to `wp-admin`

We should probably add the domain in the URL too 
eg `https://wordpress.com/pages/DOMAIN`

Generally the URLs need some auditing. Happy to help if you need!

Reply:
I think the whole data now needs updating including the URLs. Happy to have help here.

Reply:
I can have a look, if you don't have already